ui/packages/catalog-ui-search/src/main/webapp/component/query-template-sharing/query-template-sharing.hbs

```diff
@@ -22,6 +22,11 @@
 
         <div class="template-sharing-by-email"></div>
     </div>
+    <div class="information-admin">
+        <div class="is-header">By Administrators:</div>
```

Comment:
Idk about the verbiage here. Administrators and "Email", which is actually individuals, will always both have the same subject identifier.. Wonder if middle column should just say `Individuals` and right far right column should say `Administrators`.